pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -204,6 +204,9 @@ self: super: {
   # Too strict bounds on optparse-applicative
   weeder = lib.warnIf (lib.versionAtLeast super.weeder.version "2.8.0") "jailbreak on weeder may be obsolete" doJailbreak super.weeder;
 
+  # test dependency has incorrect upper bound but still supports the newer dependency
```

Comment:
Our general criteria for these comments is: How can we tell when it might be okay to remove this jailbreak. We do inevitably, but try to avoid, accumulating more and more potentially obsolete overrides in this file.

Reply:
Makes sense, I guess I’ll open an issue on the fused-effects repo